airbyte-integrations/connectors/source-linkedin-ads/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_linkedin_ads"
 
 [tool.poetry.dependencies]
 python = "^3.10,<3.12"
-airbyte-cdk = { version = "6.12.1.dev0", allow-prereleases = true } # Set this to an official release as soon as https://github.com/airbytehq/airbyte-python-cdk/pull/177 is merged and included in a CDK release.
+airbyte-cdk = "^6"
```

Comment:
Finally.

Reply:
@lazebnyi  remember you called out we emit too much state, and that this update should reduce runtime of the connector? Should this be like 1 hour instead of 10 hours, or 8 hours instead of 10 hours? Because so far we've seen the latter ;)